src/sentry/integrations/vercel/uihook.py

```diff
@@ -0,0 +1,50 @@
+from __future__ import absolute_import
+
+import json
+
+from django.http import HttpResponse, HttpResponseNotFound
+from django.views.decorators.csrf import csrf_exempt
+
+from sentry.api.base import Endpoint
+from sentry.constants import ObjectStatus
+from sentry.models import Integration, Organization, OrganizationIntegration
+from sentry.web.api import allow_cors_options
+from sentry.web.helpers import render_to_response
+
+
+class VercelUIHook(Endpoint):
+    auth_required = False
+
+    @csrf_exempt
+    @allow_cors_options
+    def dispatch(self, request):
+        if request.method == "OPTIONS":
+            return HttpResponse(status=200)
+        if request.method != "POST":
+            return HttpResponseNotFound("Page not found")
```

Comment:
@ceorourke We should really return a `405` error code here which means the method is not allowed.

Reply:
@scefali  I was just copying their Node helper library logic: https://github.com/vercel/integration-utils/blob/master/src/index.ts#L68 but I can change it